hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -29,6 +34,25 @@
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
     @Test
+    public void testMapStatsAreNotLeaking() throws Exception {
```

Comment:
Afaik, when getLocalMapStats() is called, statistics object is re-created so it won't test the actual scenario.